internal/engine/postgresql/parse_disabled.go

```diff
@@ -1,5 +1,5 @@
-//go:build windows
-// +build windows
+//go:build windows || !cgo
```

Comment:
Had to rename this file since `_windows` was implicitly causing the file to only be used on windows